code/modules/awaymissions/maploader/reader.dm

```diff
@@ -41,8 +41,8 @@ var/global/dmm_suite/preloader/_preloader = new
 	var/list/grid_models = list()
 	var/key_len = 0
 
-	dmmRegex.next = 1
-	while(dmmRegex.Find(tfile, dmmRegex.next))
+	var/regex_index = 1
+	while(dmmRegex.Find(tfile, regex_index))
```

Comment:
I know my understanding of the map reader is very limited, and I am probably wrong, but, do you ever increment the value of `regex_index`?